networkx/readwrite/gml.py

```diff
@@ -683,9 +683,12 @@ def write_gml(G, path, stringizer=None):
         .bz2 will be compressed.
 
     stringizer : callable, optional
-        A stringizer which converts non-int/float/dict values into strings. If
-        it cannot convert a value into a string, it should raise a
-        ``ValueError`` raised to indicate that. Default value: ``None``.
+        A stringizer which converts non-int/non-float/non-dict values into
+        strings. If it cannot convert a value into a string, it should raise a
+        ``ValueError`` to indicate that. Default value: ``None``.
+
+        Currently, there is no available stringizer to convert
+        non-int/non-float/non-dict values to strings.
```

Comment:
I suspect this last statement will end up being more confusing than helpful. I'd prefer to leave it out and discuss this point in greater detail in the user documentation (it's probably not appropriate for a docstring). 

There are stringizers available and they will work just fine depending on your use cases. There are not, however, stringizers/destringizer pairs that will guarantee roundtrips across all versions of Python.